types/underscore/index.d.ts

```diff
@@ -3721,14 +3721,14 @@ declare module _ {
          * @returns The item on the `object` or the `defaultValue`
          **/
         get<V extends Collection<any>>(
-            object: V,
+            object: V | null | undefined,
             path: string | string[]
         ): TypeOfCollection<V> | undefined;
         get<V extends Collection<any>, U>(
-            object: V,
+            object: V | null | undefined,
             path: string | string[],
             defaultValue?: U
-        ): TypeOfCollection<V> | U;
+        ): TypeOfCollection<V> | U | undefined;
```

Comment:
This doesn't seem right to me. Even if `object` is `null` or `undefined`, `get` will still return `defaultValue`. See https://underscorejs.org/docs/modules/get.html.